src/builders.ts

```diff
@@ -8,12 +8,45 @@ import { StacksPrivateKey } from './keys';
 
 import { TransactionSigner } from './signer';
 
-import { TransactionVersion, AddressHashMode } from './constants';
+import {
+  PostCondition,
+  STXPostCondition,
+  FungiblePostCondition,
+  NonFungiblePostCondition,
+} from './postcondition';
+
+import {
+  TransactionVersion,
+  AddressHashMode,
+  FungibleConditionCode,
+  NonFungibleConditionCode,
+} from './constants';
+
+import { StandardPrincipal, ContractPrincipal, AssetInfo } from './types';
 
 import { ClarityValue } from './clarity/clarityTypes';
 
 import * as BigNum from 'bn.js';
 
+/**
+ * STX token transfer transaction options
+ *
+ * @param  {BigNum} nonce - a nonce must be increased monotonically with each new transaction
+ * @param  {TransactionVersion} version - can be set to mainnet or testnet
+ * @param  {String} memo - an arbitrary string to include with the transaction, must be less than
+ *                          34 bytes
+ * @param  {Array<PostCondition>} postConditions - an array of post conditions to add to the
+ *                                                  transaction
+ *
+ * @return {StacksTransaction}
+ */
+export interface TokenTransferOptions {
+  nonce?: BigNum;
+  version?: TransactionVersion;
+  memo?: string;
+  postConditions?: Array<PostCondition>;
```

Comment:
Generally I've seen the shorthand array type notation used more often, would be great to stick to one.

Reply:
Is there a way to specify this in the prettier config?

Reply:
No, but we could add a lint rule to the config hank made